librz/core/cmd/cmd_shell.c

```diff
@@ -391,12 +339,9 @@ RZ_IPI RzCmdStatus rz_cmd_shell_uname_handler(RzCore *core, int argc, const char
 RZ_IPI RzCmdStatus rz_cmd_shell_echo_handler(RzCore *core, int argc, const char **argv) {
 	if (argc >= 2) {
 		char *output = rz_str_array_join(argv + 1, argc - 1, " ");
-		// TODO: replace all ${flagname} by its value in hexa
-		char *newmsg = filterFlags(core, output);
-		rz_str_unescape(newmsg);
-		rz_cons_print(newmsg);
+		rz_str_unescape(output);
```

Comment:
```suggestion
		rz_str_escape_utf8(output, <escape_options>);
```
Please could you also mark `rz_str_unescape` as deprecated while you are at it?
You could add a `const RzStrEscOptions default_unescape_ascii` options somewhere, which result to the same outcome as the deprecated function.

Reply:
Unescaping is the opposite of escaping so the suggested change doesn't make much sense. I see that `rz_str_escape` has already been marked `RZ_DEPRECATE`. Sorry I'm not ready to declare `rz_str_unescape` as deprecated just yet, since one needs to check every use of that function.

Reply:
Sorry, where is my head. Should sleep more I guess.